oluwajuwon/Fast-Food-Fast
# Setup app to use static directory with frontend templates

### What does this pull request do?
This pull request sets up the app to use static files from the UI templates

### Description of the Task to be completed?
Users should have a frontend that can be used to make requests to the backend

### How should this be manually tested?
Users can test it by:
 - Install git
 - Open up the command line
 - Run `git clone https://github.com/oluwajuwon/Fast-Food-Fast.git` to clone the repository
 - Run `npm install` to install all package dependencies
 - Run `npm run start` to start the app
 - Visit http://localhost:3000
 - It should render index.html file and should be able to navigate through other html files

### What are the relevant pivotal tracker stories?
The relevant pivotal story is:

Setup app to use static files - https://www.pivotaltracker.com/story/show/161203627

### Screenshots
![1](https://user-images.githubusercontent.com/29441898/46139882-9569b700-c247-11e8-9f8a-9046b4be1bf3.PNG)
![3](https://user-images.githubusercontent.com/29441898/46139888-97cc1100-c247-11e8-9009-e37f6b994cf0.PNG)
![4](https://user-images.githubusercontent.com/29441898/46139911-a1557900-c247-11e8-925a-a18cd5b4b5b6.PNG)
![5](https://user-images.githubusercontent.com/29441898/46139916-a3b7d300-c247-11e8-8dd0-4e7c30f42248.PNG)
